specs/passenger_spec.rb

```diff
@@ -75,4 +75,54 @@
       end
     end
   end
+
+  describe "passenger_spents method" do
+    before do
+      @trips = [RideShare::Trip.new({rating: 3, cost: 10}), RideShare::Trip.new({rating: 5, cost: 10})]
```

Comment:
For these wave 1 methods, you should include a test for what happens when the passenger has an incomplete trip.